.github/workflows/ci.yml

```
@@ -225,13 +225,15 @@ jobs:
           - 5432:5432
         options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
       minio:
-        image: minio/minio:edge-cicd
+        image: minio/minio:RELEASE.2025-10-15T17-29-55Z
         ports:
           - 9000:9000
+        command: server /data
```

actionlint (1.7.12)

[error] unexpected key "command" for "services" section. expected one of "credentials", "env", "image", "options", "ports", "volumes"

(syntax-check)